Watchman.Discord/Program.cs

```diff
@@ -18,28 +21,45 @@ public static async Task Main(string[] args)
             var configPath = "config-prod.json";
 #endif
             var configuration = JsonConvert.DeserializeObject<DiscordConfiguration>(File.ReadAllText(configPath));
-            
+
             var watchman = new WatchmanBot(configuration);
             var workflowBuilder = watchman.GetWorkflowBuilder();
-            var lastException = DateTime.MinValue;
-            var numberOfExceptionsInLastTime = 0;
 
-            while (numberOfExceptionsInLastTime++ < HOW_MANY_EXCEPTIONS_IN_SHORT_TIME_TO_STOP_BOT)
+            AppDomain.CurrentDomain.UnhandledException += (sender, e) =>
```

Comment:
pamiętaj że odpalając bota z poziomu aplikacji webowej nie bierzemy pod uwage klasy Program

Reply:
to przy okazji przygotowania dockera pod webową wersję poprawię

Reply:
Ok, zrób z tym taska i przypisz do siebie, żeby nie zapomnieć